src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -265,9 +266,22 @@ private static string GetQuotedScriptPath(string scriptPath)
             const string quote = "\"";
             // Quote the script path in case there are any spaces in it.
             // See #2347
+
+            var defaultParameters = GetDefaultParameters();
+
+
             Debug.Assert(!scriptPath.Contains(quote), "Not expecting the imported script path to be quoted");
 
-            return quote + scriptPath + quote;
+            return quote + scriptPath + quote + defaultParameters;
+        }
+
+        private static string GetDefaultParameters()
+        {
+            var workDir = Path.Combine(Path.GetTempPath(), "SLVS", "ESLintBridge", "workdir");
```

Comment:
Perhaps we need a shared helper function for returning temporary working paths e.g. 
`static GetPerVSInstanceTempDir(string featureName)`